factory_bot_rails.gemspec

```diff
@@ -1,20 +1,20 @@
 Gem::Specification.new do |s|
-  s.name        = %q{factory_bot_rails}
-  s.version     = '4.11.1'
+  s.name        = "factory_bot_rails"
+  s.version     = "4.11.1"
   s.authors     = ["Joe Ferris"]
-  s.email       = %q{jferris@thoughtbot.com}
+  s.email       = "jferris@thoughtbot.com"
   s.homepage    = "https://github.com/thoughtbot/factory_bot_rails"
-  s.summary     = %q{factory_bot_rails provides integration between
-    factory_bot and rails 3 or newer}
-  s.description = %q{factory_bot_rails provides integration between
+  s.summary     = "factory_bot_rails provides integration between
```

Comment:
The newlines in the summary and description seem funny to me. While you are here, what do you think about changing to something more like:

```
s.summary = "factory_bot_rails provides integration between "\
            "factory_bot and rails 3 or newer"
```

Reply:
Good call @composerinteralia, updated here: https://github.com/thoughtbot/factory_bot_rails/pull/295/commits/1ef282c8999d766f476dc7e7a12f3de8b223d509